### PR TITLE
THIS IS NOT A NAV BAR THING

### PR DESCRIPTION
**Pull request by bonattt**
The ListAllProposals page can edit all displayed fields in existing proposals, as long as a user is logged in as an officer.

This branch is no longer being used for the navbar. Another developer started a feature on the wrong branch, and I am finishing it.

### CODE REVIEW

File: Senior Project/html/listAllProposals.html [hunk 1]
```diff
@@ -16,47 +16,96 @@ <h1 id="signUps">Proposals</h1>
...
-                        <label>Signups Open Date</label> <br/>
-                    </div> <br/>
-					
-                    <div id="signups-modal-event_signup_close"> 
-                        <label>Signups Close Date</label> <br/>
-                    </div> <br/>
-                    <label for="imageFile">Image<input type="file" name="imageFile" id="imageFile"></label>
+					<input id="proposalModal-proposal_name" type="text" value="name"> </input><br/><br/>
+
+					<label>Monsy Allocated</label> <br/>
```
Comment (cookmn) [inline Senior Project/html/listAllProposals.html]: This should say money

File: Senior Project/html/listAllProposals.html [hunk 2]
```diff
@@ -16,47 +16,96 @@ <h1 id="signUps">Proposals</h1>
...
+					<input id="proposalModal-money_allocated" type="number" step="0.01"> </input><br/><br/>
+
+					<label>Monsy Requested</label> <br/>
```
Comment (cookmn) [inline Senior Project/html/listAllProposals.html]: This also should say money.

File: Senior Project/html/listAllProposals.html (hunk 1 above)
Comment (miskowbs) [reply Senior Project/html/listAllProposals.html]: That it should indeed

File: Senior Project/html/listAllProposals.html (hunk 2 above)
Comment (miskowbs) [reply Senior Project/html/listAllProposals.html]: Yes

**Review by miskowbs — COMMENTED** [review]
Please change Monsy to Money then we can pull

**Pull request merged**